### DISCUSSION TITLE
Winmerge in win 11

### DISCUSSION THREAD

**Discussion by windsurfer51** [WinMerge/winmerge#1910] [General]
I just installed winmerge on win 11 and when i select 2 folders to compare it only shows the first folder i selected on the left but it won't show the 2nd folder on the right.
The file compare window shows correctly.
Is this because of windows 11 ?

**Comment by sdottaka** [comment]
It's not because it's Windows 11.
WinMerge's folder comparison window displays information for both the first and second folders in one pane.

**Comment by windsurfer51** [comment]
Here is a screenshot of 2 folders i set up called temp 1 and temp 2
It only shows the contents of temp 1, and yes there is something in both folders.
https://prnt.sc/TQknrTNSptHK
![Screenshot_1](https://github.com/WinMerge/winmerge/assets/73485384/2a885124-c60e-4d15-8493-efe87665d260)

**Reply by sdottaka** [reply]
Notice the Comparison result, Left Date, Right Date columns and icons in that attached image. 
Screenshot_1.jpg is present in both D:\Temp\1 folder and D:\Temp\2, indicating that the contents match.
Screenshot_6.jpg shows that only D:\Temp\1 folder exists.
In the Filename column, the left and right filenames are displayed in this column because the filenames in the left and right folders are the same.
